docs/developer/testing.md

```diff
@@ -0,0 +1,188 @@
+# Testing guidance
+
+Prior to reviewing with key stakeholders, it's recommended to perform this series of tests to build the confidence that you've met your customers' expectations.
```

Comment:
Who are key stakeholders and where is that defined? Is this for testing within FAST?

Reply:
changed to better fit content.